features/0023-did-exchange/README.md

```diff
@@ -339,6 +352,7 @@ The invitation's `recipientKeys` should be dedicated to envelopes authenticated
 * The `did_doc~attach` optional, contains the DID Doc associated with the `did`, if required.
   * If the `did` is resolvable (either an inline `peer:did` or a publicly resolvable DID), the `did_doc~attach` attribute should not be included.
   * If the DID is a `did:peer` identifier, the DIDDoc must be as outlined in [RFC 0627 Static Peer DIDs](../0627-static-peer-dids/README.md).
+* The `did_rotate~attach` attribute is optional, but SHOULD be included if the `did` attribute is resolvable and the `did_doc~attach` is not included. The value is the Base64url encoded DID, and signed with the key used in the invitation.
 
```

Comment:
I think we should include that if a did is used in the response message that is different from the invitation, the `did_rotate~attach` MUST be present.

Reply:
Great wording, though this implies a stronger MUST requirement than we usually allow for a minor version change.

Reply:
The MUST is in the eye of the recipient. If they choose to reject a connection because its not included, then so be it.

Reply:
need more language - feels wrong (warren) to not have a MUST included.